Resources/Prototypes/Entities/Objects/Weapons/Guns/pneumatic_cannon.yml

```
@@ -14,9 +14,11 @@
   - type: Item
     size: Huge
   - type: Clothing
+    sprite: Objects/Weapons/Guns/Cannons/pneumatic_cannon.rsi #DeltaV
```

```suggestion
    sprite: Objects/Weapons/Guns/Cannons/pneumatic_cannon.rsi # DeltaV
```
Keep the formatting consistent. There is always a space between the hashtag and DeltaV